### PR TITLE
champion cards with curated content rather than from a feed

### PR DESCRIPTION
**Pull request by shsteimer**
Allow champion cards to accept curated content.

Test URLs:
- Before: https://main--theplayers--hlxsites.hlx.live/past-champions
- After: https://champion-cards-curated--theplayers--hlxsites.hlx.live/past-champions
- After: https://champion-cards-curated--theplayers--hlxsites.hlx.page/drafts/fkakatie/sentry-past-champions
- After: https://champion-cards-curated--theplayers--hlxsites.hlx.page/drafts/fkakatie/wgc-past-champions

### CODE REVIEW

File: blocks/cards/cards.js [hunk 1]
```diff
@@ -1,6 +1,15 @@
 import { createOptimizedPicture, readBlockConfig, lookupPages } from '../../scripts/scripts.js';
 
-function decorateChampionCards(champions, block) {
+function decorateChampionCards(block) {
+  [...block.children].forEach((row) => {
+    const children = row.querySelectorAll('div');
+    const pars = children[1].querySelectorAll('p');
+    pars[0].classList.add('cards-card-bubble');
+    pars[1].classList.add('cards-card-country');
```
Comment (fkakatie) [inline blocks/cards/cards.js:8]: this "country" field is optional (does not appear on sentry) and breaks if it's not there. i've got both examples in drafts to compare:
* [sentry](https://champion-cards-curated--theplayers--hlxsites.hlx.page/drafts/fkakatie/sentry-past-champions)
* [wgc](https://champion-cards-curated--theplayers--hlxsites.hlx.page/drafts/fkakatie/wgc-past-champions)

File: blocks/cards/cards.js [hunk 2]
```diff
@@ -43,6 +44,12 @@ export default async function decorate(block) {
         const bubble = div.querySelector('u');
         if (bubble) {
           bubble.className = 'cards-card-bubble';
+          bubble.closest('p').className = 'cards-card-bubble-wrapper';
+        }
+        const country = div.querySelector('.icon[class*=icon-flag-]');
+        if (country) {
+          country.closest('p').classList.add('cards-card-country');
+          div.classList.add('cards-card-country-wrapper');
```
Comment (fkakatie) [inline blocks/cards/cards.js:52]: leveraging and mirroring existing bubble logic (bubble is not unique to "champion" variant)

File: blocks/cards/cards.css [hunk 3]
```diff
@@ -174,19 +215,19 @@ main .cards.overlay > ul > li p.button-container a {
...
 @media (min-width: 900px) {
-  main .cards.cards-champions > ul {
+  main .cards.champions > ul {
     grid-template-columns: repeat(auto-fill, minmax(calc((100% / 2) - 30px), 1fr));
     grid-gap: 30px;
   }
 }
 
 @media (min-width: 700px) {
-  main .cards.cards-champions > ul > li img {
```
Comment (fkakatie) [inline blocks/cards/cards.css:189]: simplifying variant name for authors ("champion" vs. "cards-champion"), saving a whole six characters

File: blocks/cards/cards.css [hunk 4]
```diff
@@ -72,6 +97,22 @@ main .cards .cards-card-bubble {
     border-radius: 40px;
     font-size: var(--body-font-size-l);
   }
+
+  main .cards .cards-card-country-wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-end;
```
Comment (fkakatie) [inline blocks/cards/cards.css:104]: country only appears to wrap on tablet+ screen sizes

**Comment by fkakatie** [comment]
and... the more i think about it, the "champions" variant/type isn't doing anything except making the card feed two columns instead of three. it originated as "champions" because that is currently the only place it appears, but would it make more sense then to change it from champions to "two-col" or "col-2"?

**Comment by shsteimer** [comment]
Think this is good now. Players version had one issue with the wrapper missing for the bubble when the data came form the feed, but was an easy fix. Also adjusted to use `two-cols` rather than `champions` per your comment.

Maybe take one more look when you get a chance, but otherwise think this is good to merge.

**Pull request merged**